src/daily-auths-report.tsx

```diff
@@ -258,6 +258,30 @@ function DailyAuthsReport(): VNode {
 
   return (
     <div>
+      <h2>Daily Auths Report</h2>
```

Comment:
I'd consider moving this up the page and using `<h1>`. Since there's currently no h1, it causes a few accessibility issues:

- https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one
- https://dequeuniversity.com/rules/axe/4.2/heading-order

Reply:
```suggestion
      <h1>Daily Auths Report</h1>
```

Reply:
Just saw this comment in the image fixing PR, will just remove this here: https://github.com/18F/identity-reporting/pull/26/files#r710159268